### PR TITLE
[BACKLOG-1561] - Add the ability to get security/ACL information about a...

### PR DESCRIPTION
**Pull request by aliakseihaidukou**
... datasource

add get/set methods to AclAware interfaces and realizations

@pentaho-nbaker this PR is for https://github.com/pentaho/data-access/pull/592

Diogo offered to add one more interface with common methods for IAclAwareMondrianCatalogService and IAclAwarePentahoMetadataDomainRepositoryImporter. Do you think we need to do this?

### CODE REVIEW

**Comment by pentaho-nbaker** [comment]
I would not object to a common Interface between the two, if it's a good fit. I don't want to overly complicate things. It's not needed for now though.

File: extensions/src/org/pentaho/platform/plugin/action/mondrian/catalog/IAclAwareMondrianCatalogService.java [hunk 1]
```diff
@@ -31,4 +31,8 @@ void addCatalog( InputStream inputStream, MondrianCatalog catalog, boolean overw
    * @return    an ACL helper instance
    */
   IAclNodeHelper getAclHelper();
```
Comment (pentaho-nbaker) [inline extensions/src/org/pentaho/platform/plugin/action/mondrian/catalog/IAclAwareMondrianCatalogService.java]: I would like to not have this method if possible.

File: extensions/src/org/pentaho/platform/plugin/services/metadata/IAclAwarePentahoMetadataDomainRepositoryImporter.java [hunk 2]
```diff
@@ -27,4 +27,8 @@ void storeDomain( InputStream inputStream, String domainId, boolean overwrite, R
   IAclNodeHelper getAclHelper();
```
Comment (pentaho-nbaker) [inline extensions/src/org/pentaho/platform/plugin/services/metadata/IAclAwarePentahoMetadataDomainRepositoryImporter.java]: Same, would like to not have getAclHelper()

File: extensions/src/org/pentaho/platform/plugin/services/metadata/IAclAwarePentahoMetadataDomainRepositoryImporter.java [hunk 3]
```diff
@@ -27,4 +27,8 @@ void storeDomain( InputStream inputStream, String domainId, boolean overwrite, R
...
 
   RepositoryFile getDomainRepositoryFile( String domainId );
```
Comment (pentaho-nbaker) [inline extensions/src/org/pentaho/platform/plugin/services/metadata/IAclAwarePentahoMetadataDomainRepositoryImporter.java]: Who needs this? If possible, I would avoid adding this method

**Comment by aliakseihaidukou** [comment]
@pentaho-nbaker I removed those methods from interfaces. No changes to data-access needed, they were used only inside implementations.

**Pull request merged**